### ISSUE TITLE
kyma landscapes failing with intermittent timeout

### ISSUE THREAD

**Issue by kyrylo-kochubyey** [kyma-project/kyma#18635]
**Description**

All our kyma landscapes failing with intermittent timeout
Unable to connect to the server: dial tcp 20.252.121.120:443: i/o timeout
Run any kubectl commnd or call service health checks
It fails with time out, and blocks development

It fails from local and from azure agents in internent and selfhosted agents in sap network

garden-kyma--c-86a4c4e-external
https://dashboard.kyma.cloud.sap/cluster/garden-kyma--c-86a4c4e-external/overview

Metadata
Kubernetes Version:
v1.27.10
Kyma Version:
2.20.0
Storage Type:
SESSION STORAGE
API Server Address:
https://api.c-86a4c4e.kyma.ondemand.com
Provider::
AZURE

**Expected result**

We should not have timeout errors to unblock development

**Actual result**

Unable to connect to the server: dial tcp 20.252.121.120:443: i/o timeout
* connect to 20.29.178.121 port 443 failed: Connection timed out

**Steps to reproduce**
kubectl get all
Unable to connect to the server: dial tcp 20.252.121.120:443: i/o timeout

+ curl -skv https://portal.int.bnlab.cloud.ariba.com/tpx/saml-bridge/actuator/health
*   Trying 20.29.178.121:443...
* TCP_NODELAY set
* connect to 20.29.178.121 port 443 failed: Connection timed out

**Comment by kyrylo-kochubyey** [comment]
Issue with connection resolved now around 11:35AM PST

we faced this issue from 04.03.23 22:00 PM till 04.04.23 11:35AM PST

Please update us with details?

**Comment by lautoma** [comment]
Hello @kyrylo-kochubyey,

I see this issue is related to commercial service SAP BTP, Kyma Runtime. I also see that you are an SAP employee.
Such issues should not be created on the Kyma Open Source Github. There is a clear support process for the commercial offering, described for internal customers [here](https://workzone.one.int.sap/site#workzone-viewAll?sap-app-origin-hint=&/groups/QrsMLb8Me6YjlNkgcm5Ku4/overview_page/VRnBlvxxyXuFny67khRFmB). 

Please create a new issue using the right channels.

Tanks,
Laurentiu
